packages/openneuro-app/src/scripts/utils/__tests__/__snapshots__/markdown.spec.tsx.snap

```diff
@@ -2,56 +2,68 @@
 
 exports[`Test <Markdown> component > allows a href with certain protocols 1`] = `
 <DocumentFragment>
-  <a
-    href="https://example.com"
-  >
-    Example link that should work.
-  </a>
+  <p>
+    <a
+      href="https://example.com"
+    >
+      Example link that should work.
+    </a>
+  </p>
 </DocumentFragment>
 `;
 
 exports[`Test <Markdown> component > does not allow href with unknown protocols 1`] = `
 <DocumentFragment>
-  <a>
-    Example link that should not work.
-  </a>
+  <p>
+    <a>
+      Example link that should not work.
+    </a>
+  </p>
 </DocumentFragment>
 `;
 
 exports[`Test <Markdown> component > filters close-break tags 1`] = `
 <DocumentFragment>
-  <span>
+  <p>
     <br />
     sample text
     <br />
-  </span>
+  </p>
 </DocumentFragment>
 `;
 
 exports[`Test <Markdown> component > filters out disallowed tags 1`] = `
 <DocumentFragment>
   <ul>
+    
+
     <li>
       Markdown document
     </li>
+    
+
   </ul>
 </DocumentFragment>
 `;
 
 exports[`Test <Markdown> component > safely handles broken HTML tags 1`] = `
 <DocumentFragment>
   <ul>
+    
+
     <li>
       Markdown document
 
       <br />
       <br />
-      <ul>
-        <li>
-          test content
-        </li>
-      </ul>
     </li>
+    
+
+    <li>
+      test content
+    </li>
```

Comment:
This seems to lose a level of nesting. Is this a reasonable interpretation of the input markdown?

Reply:
This is the input.
```
* Markdown document
<br><br>
 * test content
```
react-markdown is more accurate to GitHub's behavior (not nesting) since the second `*` is not indented as much as the content of the first list item.